### DISCUSSION TITLE
How to start Marqo database without using Docker image or Docker compose, using pip install or apt install

### DISCUSSION THREAD

**Discussion by nikhilreddy-pottanig** [marqo-ai/marqo#934] [Q&A] [answered]
Someone help me to start this vector database

**Comment by pandu-k** [comment]
Docker is the easiest way to use Marqo. Otherwise you can follow the [developer guide](https://github.com/marqo-ai/marqo/blob/mainline/src/marqo/README.md) to run it outside of Docker, but it is trickier.

Why are you looking to start Marqo without Docker?

**Comment by nikhilreddy-pottanig** [comment]
Because Cluster I am working on doesn't support more than one image to be provided and for my requirment base image is python:3.11 and on top of that using any commands I can install any vector database. But not able to find any proper way to do it.

So can someone help me here!

**Reply by pandu-k** [reply, marked as answer]
Hi @nikhilreddy-pottanig your best bet, to run Marqo outside of Docker, is to follow the developer guide then: https://github.com/marqo-ai/marqo/blob/mainline/src/marqo/README.md#running-marqo-locally-outside-of-docker-for-development 

You can also clone the repo and modify the dockerfile so that it meets your requirements: https://github.com/marqo-ai/marqo/blob/mainline/Dockerfile

If you get stuck, post a message on our community slack.
